reference/promise-types/access.markdown

```diff
@@ -304,11 +306,15 @@ policy. Example:
 bundle server my_access_rules()
 {
 access:
-
   "/directory/"
-
-    admit   => { "127.0.0.1", ".example.org" },
-    deny    => { "badhost_1.example.org", "badhost_1.example.org" };
+    admit => {
+      "127.0.0.1",
+      ".example.org"
```

Comment:
Maybe use trailing comma in wrapped lists?

Reply:
Fixed, thanks.